tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -15565,6 +15565,210 @@ TEST_F(ZoneUnitarySysTest, UnitarySystemModel_SetpointControlCyclingFan)
     EXPECT_TRUE(compare_err_stream(error_string, true));
 }
 
+TEST_F(ZoneUnitarySysTest, UnitarySystemModel_ScheduleInterpolationTest)
```

Comment:
This unit test is based on the test before it: ([link](https://github.com/NREL/EnergyPlus/blob/f84f89f6d6cb935885ca7af3fea3664fe6f6516b/tst/EnergyPlus/unit/UnitarySystem.unit.cc#L15412))

Reply:
@jmythms It would be better for this test to focus only on the schedule without the extra baggage of a UnitarySystem. In fact, there are several existing unit tests for Schedule:Day:Interval in ScheduleManager.unit.cc `ScheduleDayInterval*`. I would either extend one of these (or copy one to make a new unit test) to specifically trip on this issue.